docs/demo/wasm2wat/demo.js

```diff
@@ -14,7 +14,17 @@
  * limitations under the License.
  */
 
-var features = {};
+
+var features = getLocalStorageFeatures()
+
+function getLocalStorageFeatures() {
+  try {
+    return JSON.parse(localStorage && localStorage.getItem("features")) || { mutable_globals: true };
```

Comment:
Can you factor out the `{mutable_globals:true}` (here and below) into a separate `defaultFeatures`?